src/main/kotlin/traffic_simulation/RoadNetwork.kt

```diff
@@ -2,33 +2,50 @@ package traffic_simulation
 
 class RoadNetwork(val capacity: Int) {
 
-    fun calculateDemand(vehicles: List<Vehicle>): Int {
-        var capacityDemand: Int = 0
+    fun calculateDemandFor24Hours(vehicles: List<Vehicle>): MutableMap<Int , Int> {
```

Comment:
Das wäre recht schade. Dann müssten auch wieder die Tests geändert werden. Kann das nicht @jwustmann in seiner Issue #66 so anpassen wie er es schließlich auch braucht?